examples/chat/build.gradle

```diff
@@ -1,5 +1,5 @@
 apply from: "../examples.gradle"
 
 dependencies {
-    implementation("com.graphql-java:graphql-java-extended-scalars:1.0.1")
+    implementation("com.graphql-java:graphql-java-extended-scalars:2020-02-19T04-04-26-6f82be9")
```

Comment:
@marceloverdijk is it ok to update this? I'm not sure if this new version contains any breaking change.

Reply:
@ilopmar I wonder if that is an actual released version? I also don't see it on Maven Central: https://search.maven.org/artifact/com.graphql-java/graphql-java-extended-scalars

Reply:
It is in JCenter: https://jcenter.bintray.com/com/graphql-java/graphql-java-extended-scalars/

To avoid this from happening again in the future, you need to pin the dependency to the version you want:

```groovy
micronautBuild {
    resolutionStrategy {
        force "com.graphql-java:graphql-java-extended-scalars:1.0.1"
    }    
}
```

Reply:
Thank you @alvarosanchez, I'll make the changes.